_includes/components/search-results/SearchResult/partials/normaliseReturn.js

```diff
@@ -0,0 +1,10 @@
+import normaliseItem from './normaliseItem.js';
+
+
+export default function normaliseReturn(returnObj) {
```

Comment:
I have absolutely no idea what this module does. Does it always normalise the response from package_search to the same internal form? If so, we could really call the module NormalisePackageSearchReturn or have a comment hint to the service it's about.

Reply:
This and `normaliseItem` can actually be merged into one module. In short, that would then just be a module that takes the AJAX response and turns it into a object that's more semantic to the front-end/UI. For example:

![image](https://user-images.githubusercontent.com/14258328/39930351-c810706a-553a-11e8-9602-53c01fa5e08e.png)

Reply:
Renamed the module to `normaliseServerResponse`